deepset_cloud_sdk/workflows/async_client/files.py

```diff
@@ -43,6 +43,7 @@ async def list_files(
     :param name: Name of the file to odata_filter for.
     :param content: Content of the file to odata_filter for.
     :param odata_filter: The odata_filter to apply to the file list.
+    For example, `odata_filter="category eq 'news'' lists files with metadata `{"meta": {"category": "news"}}.
```

Comment:
There is something wrong with the quotes. The example with the CLI has ' for outer and " for inner whereas this starts with " for outer but then ends with '

Reply:
the CLI one worked for me, this one worked for Sebastian :) I'll double check